dovecot/dovecot24-lib.pl

```diff
@@ -0,0 +1,68 @@
+# Dovecot 2.4 compatibility shims
+
+# params([param])
+# Simple mapping
+sub params
+{
+my $name = shift;
+my %map = (
+	# Auth
+	auth_default_realm     => 'auth_default_domain',
+	disable_plaintext_auth => 'auth_allow_cleartext',
+
+	# Mail
+	mail_location          => 'mail_path',
+
+	# SSL
+	ssl_ca                 => 'ssl_server_ca_file',
+	ssl_ca_file            => 'ssl_client_ca_file',
+	ssl_cert               => 'ssl_server_cert_file',
+	ssl_cert_file          => 'ssl_server_cert_file',
+	ssl_key                => 'ssl_server_key_file',
+	ssl_key_file           => 'ssl_server_key_file',
+	ssl_key_password       => 'ssl_server_key_password',
+	);
+return wantarray ? %map : ($map{$name} || $name);
+}
+
+# map_find(name)
+# Translate key
+sub map_find
```

Comment:
All these function names are way too short ... they should be more descriptive.

Reply:
I just wanted the function names to be simple and correspond with the original API.